src/resolve/resolved-dnssd.c

```diff
@@ -157,6 +157,7 @@ static int specifier_dnssd_host_name(char specifier, const void *data, const voi
 
 int dnssd_render_instance_name(DnssdService *s, char **ret_name) {
         static const Specifier specifier_table[] = {
+                COMMON_SYSTEM_SPECIFIERS,
```

Comment:
Hmm? This change sounds conflict with the commit message:
> Note that some entries that look like they could be unified under these
macros can't really be unified, since they are slightly different. For
example in the DNSSD service logic we want to use the DNSSD hostname for
%H rather than the unmodified kernel one.

Maybe, use the macro in resolved-conf.c, instead of here?

Reply:
oops, that was some left-over noise. Will drop.

(I originally patched the DNSSD code too, but then I realized that it has some more complex stuff going on there, hence dropped that again, but made a mistake. We can look into porting that over too eventually, but I wanted to start out with the safe, obvious stuff)